enterprise/server/tasksize/tasksize.go

```diff
@@ -178,7 +183,10 @@ func (s *taskSizer) Update(ctx context.Context, cmd *repb.Command, md *repb.Exec
 	}
 	statusLabel := "ok"
 	defer func() {
-		props := platform.ParseProperties(&repb.ExecutionTask{Command: cmd})
+		props, err := platform.ParseProperties(&repb.ExecutionTask{Command: cmd})
+		if err != nil {
+			log.CtxWarningf(ctx, "Failed to parse task properties: %s", err)
```

Comment:
(forgot to push changes?)

```suggestion
			log.CtxInfof(ctx, "Failed to parse task properties: %s", err)
```